src/installer/tests/Directory.Build.props

```diff
@@ -4,12 +4,11 @@
   <PropertyGroup>
     <TestArtifactsOutputRoot>$(ArtifactsDir)tests\host\$(TargetOS).$(TargetArchitecture).$(Configuration)\</TestArtifactsOutputRoot>
     <TestInfraTargetFramework>$(NetCoreAppToolCurrent)</TestInfraTargetFramework>
-    <TestCaseFilter>category!=failing</TestCaseFilter>
-    <TestRunnerAdditionalArguments>--filter $(TestCaseFilter) -v detailed</TestRunnerAdditionalArguments>
+    <TestCaseFilter>/[category!=failing]</TestCaseFilter>
+    <TestRunnerAdditionalArguments>--filter-query $(TestCaseFilter)</TestRunnerAdditionalArguments>
```

Comment:
This is one example why it's not encouraged to mix VSTest and MTP.
Are you able to afford the differences? Do you have any places where `TestRunnerAdditionalArguments` has values that are shared across the whole test run, which can end up compatible with the xunit.v3 projects but not the xunit 2?

Any blockers moving the whole repo to xunit.v3, which I imagine will eventually need to happen anyways?